tv_apps/smart-clock/js/clock_component.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+(function(exports) {
+
+function ClockComponent() {}
+
+ClockComponent.prototype = {
```

Comment:
This class is suitable to be a web component. It possible, please write it as web component.